### ISSUE TITLE
Issues with the validation errors

### ISSUE THREAD

**Issue by roxigonzalez** [openactive/dataset-site-template-ruby#15]
```
FATAL ERROR: Validation errors were found in the opportunity data feeds. 657 errors were reported of which 1 were unique.
Open ./output/validation-errors.html or http://localhost:3000/validation-errors in your browser for more information

```

**Comment by roxigonzalez** [comment]
![Screenshot 2021-06-09 at 11 50 02](https://user-images.githubusercontent.com/5349378/121404079-d41cfe00-c918-11eb-85bb-ac4d8783e628.png)

**Comment by roxigonzalez** [comment]
![Captura de Pantalla 2021-06-09 a la(s) 11 41 34](https://user-images.githubusercontent.com/5349378/121404143-e39c4700-c918-11eb-83bc-ef64a5e2b6ef.png)

**Comment by nickevansuk** [comment]
The following settings are required to be added to the Ruby gem:
https://github.com/openactive/OpenActive.DatasetSite.NET/blob/a6335fd78066098d760945cb4674dca66972d5b8/OpenActive.DatasetSite.NET/models/DatasetSiteGeneratorSettings.cs#L90-L115

These need to be added to the output in the same way as in the .NET implementation:
https://github.com/openactive/OpenActive.DatasetSite.NET/blob/a6335fd78066098d760945cb4674dca66972d5b8/OpenActive.DatasetSite.NET/DatasetSiteGenerator.cs#L135-L146

The relevant Ruby file is here:
- https://github.com/openactive/dataset-site-template-ruby/blob/master/lib/openactive/dataset_site/settings.rb

If this works, the JSON-LD in the Dataset Site should include:
```
  "accessService": {
    "@type": "WebAPI",
    "name": "Open Booking API",
    "description": "API that allows for seamless booking experiences to be created for sessions and facilities available from Example",
    "authenticationAuthority": "https://auth.reference-implementation.openactive.io",
    "conformsTo": [
      "https://openactive.io/open-booking-api/EditorsDraft/"
    ],
    "documentation": "https://developer.openactive.io/go/open-booking-api",
    "endpointDescription": "https://www.openactive.io/open-booking-api/EditorsDraft/swagger.json",
    "endpointURL": "https://reference-implementation.openactive.io/api/openbooking",
    "landingPage": "https://example.com/api-landing-page",
    "termsOfService": "https://example.com/api-terms-page"
  },
```

Please note that the .NET function `dataFeedHumanisedList.ToLowerInvariant()` is the same as the Ruby function `data_feed_descriptions.to_sentence`

Please note you will need to update the `openactive` Ruby gem dependency to do the above.

**Comment by roxigonzalez** [comment]
Hey @nickevansuk 
I have this error, I'm not sure if is related to the changes that I made.
 
![Screenshot 2021-06-10 at 13 18 39](https://user-images.githubusercontent.com/5349378/121584148-6216e880-c9ee-11eb-8898-b5c50a733122.png)

**Comment by nickevansuk** [comment]
Resolved by #18
